crates/network/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
I like the idea of encapsulating all the necessary changes to add a new network in a single place. I think there are probably more types basic enough that we could have shared code for them (although this could also lead to another `shared` crate with too much stuff in there if we are not careful).

Maybe it makes sense to leave this as a separate `network` crate for now but be open to the idea to turn it into a `primitive_types` crate later on to also move `Address`, `ContractAddress`, `TokenAddress`, etc. into it.